packages/server/src/login/login-course.service.ts

```diff
@@ -51,23 +46,21 @@ export class LoginCourseService {
       }),
     );
 
-    await Promise.all(
-      info.ta_courses.map(async (c: KhouryTACourse) => {
-        // Query for all the courses which match the name of the generic course from Khoury
-        const courseMappings = await CourseSectionMappingModel.find({
-          where: { genericCourseName: c.course }, // TODO: Add semester support
-        });
+    for (const c of info.ta_courses) {
+      // Query for all the courses which match the name of the generic course from Khoury
+      const courseMappings = await CourseSectionMappingModel.find({
+        where: { genericCourseName: c.course }, // TODO: Add semester support
+      });
 
-        for (const courseMapping of courseMappings) {
-          const taCourse = await this.courseToUserCourse(
-            user.id,
-            courseMapping.courseId,
-            c.instructor === '1' ? Role.PROFESSOR : Role.TA,
-          );
-          userCourses.push(taCourse);
-        }
-      }),
-    );
+      for (const courseMapping of courseMappings) {
+        const taCourse = await this.courseToUserCourse(
+          user.id,
+          courseMapping.courseId,
+          c.instructor === 1 ? Role.PROFESSOR : Role.TA,
```

Comment:
Can we also change the type in `KhouryTACourse` to `number`?

Reply:
Just checked and cypress is failing because of this too

Reply:
wait lol I just forgot to cmmit it HAHA